prow/config/jobs.go

```diff
@@ -85,7 +85,7 @@ type JobBase struct {
 	// MaximumConcurrency of this job, 0 implies no limit.
 	MaxConcurrency int `json:"max_concurrency,omitempty"`
 	// Agent that will take care of running this job.
-	Agent string `json:"agent"`
+	Agent string `json:"agent,omitempty"`
```

Comment:
These are great -- OpenShift had caught most but we never stopped setting agent when Cole defaulted it!